src/main/java/org/folio/entlinks/controller/delegate/AuthoritySourceFileServiceDelegate.java

```diff
@@ -161,6 +164,17 @@ private void validatePatchRequest(AuthoritySourceFilePatchDto patchDto, Authorit
     }
   }
 
+  public void validateNoReferencesForSourceFile(AuthoritySourceFile sourceFile) {
```

Comment:
Can't we merge it with 'anyAuthoritiesExistForSourceFile' where we also requesting consortium tenants?
Validation also should be before deleting the sequence

Reply:
I think we can't in case differences of relation: 
'anyAuthoritiesExistForSourceFile' checks authority source files and authority
validateNoReferencesForSourceFile checks authority source files and authority archive

Oh, yeah. Thanks, I'll move this validation before sequence deletion

Reply:
I've moved validation.

Reply:
I meant reuse those lines
`var consortiumTenants = consortiumTenantsService.getConsortiumTenants(context.getTenantId());
    for (String memberTenant : consortiumTenants) {`
And do both validations in a same loop

Reply:
I think I'm done, please review.